docs/GETTING-STARTED.md

```diff
@@ -71,13 +71,19 @@ You're almost done.
 
 ## Start trading
 
-Log into your Heroku account and open the console:
+Log into your Heroku account and open your deployed app:
 
 ![heroku console](images/image1.png)
 
-Enter `npm run start` into the console and it will start the trading bot, which is now connected to both your Binance and your BVA account.
```

Comment:
Don't users have to run this command? I'm not using Heroku, so I can't tell.

Reply:
No, it’s not necessary because everytime you build a NodeJS app it will run automatically (from my experience)

Reply:
Alrighty